aerogear-android-push/src/main/java/org/jboss/aerogear/android/unifiedpush/gcm/UnifiedPushConfig.java

```diff
@@ -32,9 +32,16 @@
     private String alias;
     private List<String> categories = new ArrayList<String>();
 
+    
+    /**
+     * Topics in GCM must conform to this pattern.
+     * See : https://developers.google.com/android/reference/com/google/android/gms/gcm/GcmPubSub#unsubscribe(java.lang.String, java.lang.String)
+     */
+    private static final String GCM_TOPIC_PATTERN = "[a-zA-Z0-9-_.~%]{1,900}";
```

Comment:
Just wondering, should we require this format for our categories on the server too ?